docs/native/src/onboarding/onboarding.rst

```diff
@@ -22,67 +22,72 @@ Onboarding information
 The following information must be supplied to Microsoft to enable end-to-end testing and release of
 |Office iOS| integration.
 
-+----------------+-------------+--------------------------------------------+--------------------------------------------------+
-| Property       | Data Type   | Sample Value                               | Description                                      |
++-----------------+-------------+--------------------------------------------+--------------------------------------------------+
+| Property        | Data Type   | Sample Value                               | Description                                      |
```

Comment:
I think I fixed it.

From: Olga Dalecka [mailto:notifications@github.com]
Sent: Tuesday, June 7, 2016 3:32 PM
To: Microsoft/Office-Online-Test-Tools-and-Documentation <Office-Online-Test-Tools-and-Documentation@noreply.github.com>
Cc: Karvell Li <karvell@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [Microsoft/Office-Online-Test-Tools-and-Documentation] Update onboarding.rst (#220)


In docs/native/src/onboarding/onboarding.rst<https://github.com/Microsoft/Office-Online-Test-Tools-and-Documentation/pull/220#discussion_r66164721>:

> @@ -22,67 +22,72 @@ Onboarding information

>  The following information must be supplied to Microsoft to enable end-to-end testing and release of

>  |Office iOS| integration.

>

> -+----------------+-------------+--------------------------------------------+--------------------------------------------------+

> -| Property       | Data Type   | Sample Value                               | Description                                      |

> ++-----------------+-------------+--------------------------------------------+--------------------------------------------------+

> +| Property        | Data Type   | Sample Value                               | Description                                      |

When I go to "View", I don't see your table and I think that's because you added a '-' to this row, but you didn't extend all of the rows in this column with a '-'.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Microsoft/Office-Online-Test-Tools-and-Documentation/pull/220/files/4a692db9b9d00706edf29ad2bf820f43956b0314#r66164721>, or mute the thread<https://github.com/notifications/unsubscribe/AQKUYhbPTNblFS_zGaI4nPIIoVete5xfks5qJfFjgaJpZM4IwZh9>.